README.md

```diff
@@ -136,6 +136,20 @@ tls_dns_name = "1dot1dot1dot1.cloudflare-dns.com"
 trust_nx_responses = false
 ```
 
+## Syntax check
+To check if the config has a valid syntax, the following command can be used.
+
+`cargo run -- --dry-run`
```

Comment:
I'm not sure I like the use of `cargo run` over calling an installed command like `crab-hole` here. The readme so far only suggests installing this program, not running it in its source folder. @LuckyTurtleDev what do you think?

Reply:
You are right. This should be changed.
Due to `cargo` being the only command listed, I just went with it.
I think it might be good to add an *Running crab-hole* section, or something similar.

Should this call the installed command, likely `crab-hole`, or just mention the additional flags?